lib/Base/Option.fram

```diff
@@ -0,0 +1,25 @@
+{# This file is part of DBL, released under MIT license.
+ # See LICENSE for details.
+ #}
+
+import open /Base/Types
+import open /Base/Assert
+
+{## For `Some x` returns `x`, otherwise the provided argument is returned.
+ #}
+pub method unwrapOr default =
```

Comment:
I was wondering, if the `default` parameter should be named, because in almost all cases it will be provided. But I think, the proposed interface is good: the name of this method suggests what is the meaning of its first parameter, and writing code like `x.unwrapOr {default = 42}` would make some unnecessary noise.

Reply:
Alternative (although I don't know how good) would be for `unwrap` to take optional argument default.

Reply:
Functions `unwrap` with optional parameter and `unwrapOr` have different effects. The latter is more pure (`->` instead of `->[]`).